pkg/pb/piecestore.proto

```diff
@@ -84,7 +84,7 @@ message PieceRetrieval {
 
   RenterBandwidthAllocation bandwidth_allocation = 1;
   PieceData piece_data = 2;
-  SignedMessage authorization = 3;
+  PayerBandwidthAllocation payer_allocation = 3;
```

Comment:
`PayerBandwidthAllocation` should be inside `RenterBandwidthAllocation`. Most probably we don't need new field.

Reply:
I know we talked already but in case anyone else is reviewing--
the pieceRanger.Range function only has PBA on the struct, not RBA - so only PBA can be passed through as a message on the stream. Even though pieceRange later creates NewStreamReader which should add PBA to RBA, it doesn't seem to be working because calling `recv.GetBandwidthAllocation()` inside the Retrieve function on the server side yields nil (here and in master).
https://github.com/storj/storj/blob/877dd3deb0531025b43a481b702ee78b4ff7d0e0/pkg/piecestore/psclient/pieceranger.go#L51